server/planning/io/ingest_rule_handler.py

```diff
@@ -74,15 +74,18 @@ def apply_rule(self, rule: Dict[str, Any], ingest_item: Dict[str, Any], routing_
 
         if updates is not None:
             ingest_item.update(updates)
-
-        if attributes.get("autopost", False):
+        elif attributes.get("autopost", False):
```

Comment:
This might still need to be an `if` statement, because an item can be created, ingest rule add a calendar AND publish it, otherwise the item won't get published on ingest of first version.

Reply:
when it adds calendar it calls `.patch` on event which does the publishing